manim/animation/transform.py

```diff
@@ -229,6 +229,61 @@ def interpolate(self, alpha: float) -> None:
 
 
 class ClockwiseTransform(Transform):
+    """Transforms `Mobject`s by moving each of its points in a clockwise path to its target point.
+
+    Examples
+    --------
+
+    .. manim:: ClockwiseExample
+
+        class ClockwiseExample(Scene):
+            def construct(self):
+                dl, dr = Dot(), Dot()
+                sl, sr = Square(), Square()
+
+                VGroup(dl, sl).arrange(DOWN).shift(2*LEFT)
+                VGroup(dr, sr).arrange(DOWN).shift(2*RIGHT)
+
+                self.add(dl, dr)
+                self.wait()
+                self.play(
+                    ClockwiseTransform(dl, sl),
+                    Transform(dr, sr)
+                )
+                self.wait()
+
+    .. manim:: ClockwiseTransform_vs_Transform
```

Comment:
I'm still not convinced this example is needed, but I won't block the PR over it. In general the idea of this is fine; just two things I'm not happy with: (1) the to "vs" examples are long (in the sense of animation time) which is also reflected by the documentation render time. From the latest documentation build log:
```
animation.transform.ClockwiseTransform.......................5.056s  => 2 EXAMPLES
                                                             1.069s ClockwiseExample
                                                             3.987s ClockwiseTransform_vs_Transform
animation.transform.CounterclockwiseTransform................5.428s  => 2 EXAMPLES
                                                             1.064s CounterclockwiseExample
                                                             4.364s CounterclockwiseTransform_vs_Transform
```
These 8 seconds could be spent on 8 shorter / more concise examples.

And (2), the fact that the integers on the bottom disappear while the ones in the middle remain gives these examples a somewhat inconsistent feeling (but this is a *very* high-level and nitpicky comment 😅).

Reply:
i've reduced the time to 4 secs
I can further reduce it to 2 secs, but I think it is becoming a little clumsy

Reply:
What you could do is remove the examples `ClockwiseTransform_vs_Transform` and `CounterclockwiseExample_vs_Transform` and just have examples for those specific classes 
(i.e, `ClockwiseTransform` and `CounterclockwiseTransform`) themselves.

Maybe remove `CounterclockwiseTransform` example as well? It seems intuitive enough what it would do after seeing the clockwise transform.

Or consider putting a `ClockwiseTransform_vs_Transform` instead of `ClockwiseTransform`

Reply:
i've put `ClockwiseTransform` and `CounterclockwiseTransform_vs_Transform`

everything necessary is illustrated with these 2 examples for `Clockwise` and `Counterclockwise` while reducing the animation time

is this ok or do I have to any other changes?